nodebuilder/tests/da_test.go

```diff
@@ -105,11 +104,13 @@ func TestDaModule(t *testing.T) {
 		{
 			name: "GetIDs",
 			doFn: func(t *testing.T) {
-				t.Skip()
```

Comment:
I'm curious why it was skipped before. Anyone knows?

Reply:
IIRC order of returned IDs was non-deterministic in the past.